lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -40,9 +40,27 @@ if test "x$suse_cloud_version" != "x5"; then
 fi
 
 
+### Restart some services
+echo_summary "Restarting required services..."
+
+if test ! -f /etc/chef/amqp_passwd; then
+  grep -m1 "^amqp_pass " /etc/chef/server.rb | sed 's/amqp_pass "\([^"]*\)"/\1/g' > /etc/chef/amqp_passwd
```

Comment:
Because it have been moved to the package it really belongs to. ```rubygem-chef-server``` is just some kind of meta gem that installs the other gems as a dependency. The configuration, systemd or sysvinit files are really only used by ```rubygem-chef-server-api``` or ```rubygem-chef-server-webui``` if it gets installed.